mod/viewcontacts.php

```diff
@@ -64,6 +64,7 @@ function viewcontacts_content(&$a) {
 		$contacts[] = array(
 			'id' => $rr['id'],
 			'img_hover' => sprintf( t('Visit %s\'s profile [%s]'), $rr['name'], $rr['url']),
+			'photo_menu' => contact_photo_menu($rr),
```

Comment:
You can't use the contact_photo_menu() function at this point because not only the local_user does have access to this page. With this line even non friendica members have access to something like "edit contact"